app/scrapers/ask.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import print_function
```

Comment:
Where is this file being used?

Reply:
This was present in all the files in scrapers folder. So I didn't change them.

Reply:
Oh, I mean this whole file.

Reply:
There were separate scraper files for each of the search engines. Helps in maintenance.

Reply:
@AnshulMalik  Please look at  #141, the modularized scrapers have been updated/added as of that pull.